flow-typed/react-intl_vx.x.x.js

```diff
@@ -1,14 +1,9 @@
 // `react-intl`, translated from the relevant-looking TypeScript .d.ts
 // file, with minimal tweaks to make it work.
 
-// Anything at `React_2.` is `any`; see
-// https://chat.zulip.org/#narrow/stream/243-mobile-team/topic/libdef.3A.20react-native-webview/near/896571.
-// We should use Flow's builtin React type annotations, like
-// React$Node instead of React.Node.
-
-// Not sure why the TypeScript uses React_2 instead of just React, but
-// we've kept it intact.
-import * as React_2 from 'react';
+// This used to try to use types from React's TypeScript libdef,
+// @types/react. We've translated those places to use Flow instead,
+// hopefully without errors or too much loss of information.
```

Comment:
Thanks for doing this!

It's too bad that (as you say in the commit message) imports don't work inside Flow libdefs. That would simplify a good bit of this.

Probably an ideal version of Flowgen would work around that by copying in the relevant definitions from the referenced other libdef.